### DISCUSSION TITLE
In browsers on iPadOS, menus are stuck open

### DISCUSSION THREAD

**Discussion by cwinters8** [community/community#72828] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

While navigating github.com on my iPad using Firefox or Edge, there are menus that are stuck open, even though I never opened them to begin with. I’m unable to dismiss the menus, either by selecting the dropdown or selecting one of the options. This happens on nearly every page, including the one I’m typing this on right now. Safari seems to not have this problem. I don’t use Chrome, so I’m not sure about that one.

Here are some screenshots to show what I mean.

![cwinters8 profile](https://github.com/community/community/assets/40615752/3d9a245e-b580-4d7c-9fa0-619d8ac4c029)

![DevonFarm organization](https://github.com/community/community/assets/40615752/6e517487-cb78-42b8-b707-fa1a82938d36)

![Dashboard](https://github.com/community/community/assets/40615752/3d0a7ead-6d59-4b9d-b9b4-829d63e0df08)

![Discussion](https://github.com/community/community/assets/40615752/c41c57f5-8c1f-457a-9a44-3daf772d2b5b)

I understand there may not be a lot of people navigating GitHub on an iPad, but I sometimes use it when I have things to do and I’m not near a regular computer.

**Comment by abdulrahim-ramadan** [comment]
Response#1

**Comment by ncastellanosort** [comment]
Hello cwinter08 !

It may be a problem of initialization of the ipad, you may need to restart it, maybe the ipad has not started well some drivers of the screen, so it would be good to do that, or you can also check the ipad updates, as well as the battery status of the same, I hope it helps you!

Nicolas.
